ykman-gui/main.cpp

```diff
@@ -60,11 +61,22 @@ int main(int argc, char *argv[])
 
     engine.load(QUrl(url_prefix + main_qml));
 
-    if (argc > 2 && strcmp(argv[1], "--log-level") == 0) {
-        if (argc > 4 && strcmp(argv[3], "--log-file") == 0) {
-            QMetaObject::invokeMethod(engine.rootObjects().first(), "enableLoggingToFile", Q_ARG(QVariant, argv[2]), Q_ARG(QVariant, argv[4]));
+    QCommandLineParser parser;
+    parser.setApplicationDescription("Cross-platform application for YubiKey configuration");
```

Comment:
Add a . in the end.

Reply:
Hm. This is just a phrase, not a full sentence. But comparing with `ykman` and a few Unix toolbox programs, it seems like an imperative sentence is the norm. How about rewriting this to "Configure your YubiKey using a graphical application."?

Reply:
Sounds good!

Reply:
Done!